### PR TITLE
start VMM2020

### PR DESCRIPTION
**Pull request by wirthi**

### CODE REVIEW

File: 2020/cfp.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+                   Call for Participation: VMM'2020
+                   ================================
+
+                      6th Virtual Machine Meetup
+
+                         Co-located with MPLR
+                        as virtual online event
```
Comment (davleopo) [inline 2020/cfp.md]: as "a virtual "?

File: 2020/cfp.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+
+Topics include, but are not limited to:
+
+ - Programming language design
+ - Dynamic and static program analysis
+ - Compiler construction
+ - Managed runtime architectures
+ - Data processing engines
+ - Distributed execution environments
```
Comment (davleopo) [inline 2020/cfp.md:29]: Id add graalvm focus stuff as well like "interesting developments in the GraalVM eco system"

File: 2020/cfp.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+ - Registration for participation: October 31st, 2020
+ - Virtual machine Meetup: November 4-6th online (via Zoom)
+
+Submission link:
+
+ - http://vmmeetup.github.io/
+
+Program committee:
+
+ - David Leopoldseder, Johannes Kepler University, Linz
```
Comment (davleopo) [inline 2020/cfp.md]: labs I am ;-)

**Review by davleopo — CHANGES_REQUESTED** [review]

File: 2020/cfp.md (hunk 2 above)
Comment (wirthi) [reply 2020/cfp.md:29]: I've put ` - Developments in the GraalVM ecosystem and related topics, including` as first bullet point.

File: 2020/cfp.md (hunk 3 above)
Comment (wirthi) [reply 2020/cfp.md]: I liked the idea we had reviewers from different institutions ;-) but fixed

Comment (davleopo) [reply 2020/cfp.md]: but that would be a lie :D

**Review by davleopo — APPROVED** [review]

**Pull request merged**